modules/nf-core/crispresso2/tests/nextflow.config

```diff
@@ -0,0 +1,13 @@
+process {
+    withName: 'CRISPRESSO2' {
+        ext.args = { 
+            if (meta.id == 'nhej_test') {
+                '--quantification_window_size 10 --plot_window_size 20 --min_frequency_alleles_around_cut_to_plot 0.05'
+            } else if (meta.id == 'paired_test') {
+                '--paired_reads --min_paired_end_reads_overlap 10'
+            } else {
+                ''
+            }
+        }
+    }
+}
```

Comment:
It is better to have one `nextflow.config` file for test, and include them in the scope of the test. To avoid the if/else.